internal/service/apprunner/vpc_connector.go

```diff
@@ -167,9 +171,27 @@ func resourceVPCConnectorRead(ctx context.Context, d *schema.ResourceData, meta
 		return diag.FromErr(fmt.Errorf("error setting tags: %w", err))
 	}
 
+	if err := d.Set("tags_all", tags.RemoveDefaultConfig(defaultTagsConfig).Map()); err != nil {
+		return diag.FromErr(fmt.Errorf("error setting tags_all: %w", err))
+	}
+
 	return nil
 }
 
+func resourceVPCConnectorUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	conn := meta.(*conns.AWSClient).AppRunnerConn
+
+	if d.HasChange("tags_all") {
+		o, n := d.GetChange("tags_all")
+
+		if err := UpdateTags(conn, d.Get("arn").(string), o, n); err != nil {
+			return diag.FromErr(fmt.Errorf("error updating App Runner vpc Connector (%s) tags: %s", d.Get("arn").(string), err))
```

Comment:
Simplify to use `diag.Errorf`

```suggestion
			return diag.Errorf("error updating App Runner vpc Connector (%s) tags: %s", d.Get("arn").(string), err)